packages/gateway/src/graphql/config.ts

```diff
@@ -105,6 +116,10 @@ export const getApolloConfig = (): Config => {
     typeDefs,
     resolvers,
     introspection: true,
+    dataSources: () => ({
```

Comment:
```suggestion
    dataSources: (): Context['dataSources'] => ({
```
This will make it so that when we make changes in the future, amending the types will also become necessary

Reply:
Agreed 👍🏼 , done in https://github.com/opencrvs/opencrvs-core/pull/4706/commits/270763159f7c733bf6296823867ad7560ffc0b0e